asistente_ladm_col/gui/data_model_converter/dlg_data_model_converter.py

```diff
@@ -0,0 +1,495 @@
+                  db_pre.names.COL_RRR_SOURCE_T: None,
+                  db_pre.names.COL_UNIT_SOURCE_T: None,
+                  db_pre.names.COL_UE_SOURCE_T: None,
+                  db_pre.names.COL_BAUNIT_SOURCE_T: None}
+        conv_layers = {db_conv.names.LC_SURVEY_POINT_T: None,
+                  db_conv.names.LC_CONTROL_POINT_T: None,
+                  db_conv.names.LC_BOUNDARY_POINT_T: None,
+                  db_conv.names.LC_BOUNDARY_T: None,
+                  db_conv.names.POINT_BFS_T: None,
+                  db_conv.names.LC_PLOT_T: None,
+                  db_conv.names.LC_BUILDING_T: None,
+                  db_conv.names.LC_BUILDING_UNIT_T: None,
+                  db_conv.names.LC_RIGHT_OF_WAY_T: None,
+                  db_conv.names.COL_VALUE_AREA_T: None,
+                  db_conv.names.MORE_BFS_T: None,
+                  db_conv.names.LESS_BFS_T: None,
+                  db_conv.names.LC_TIPOLOGY_BUILDING_T: None,
+                  db_conv.names.LC_NON_CONVENTIONAL_QUALIFICATION_T: None,
+                  db_conv.names.LC_CONVENTIONAL_QUALIFICATION_T: None,
+                  db_conv.names.LC_GROUP_QUALIFICATION_T: None,
+                  db_conv.names.LC_BUILDING_OBJECT_T: None,
+                  db_conv.names.LC_PARCEL_T: None,
+                  db_conv.names.LC_SURVEY_CADASTRAL_ADITIONAL_DATA_T: None,
+                  db_conv.names.LC_PARCEL_NUMBER_NEW_STRUCTURE_T: None,
+                  db_conv.names.LC_FMI_NEW_STRUCTURE_T: None,
+                  db_conv.names.LC_CONTACT_VISIT_T: None,
+                  db_conv.names.LC_CONDOMINIUM_PH_DATA_T: None,
+                  db_conv.names.LC_REAL_ESTATE_MARKET_OFFERS_T: None,
+                  db_conv.names.EXT_ADDRESS_S: None,
+                  db_conv.names.LC_COPROPERTY_T: None,
+                  db_conv.names.BUILDING_UNIT_CHARACTERISTICS_T: None,
+                  db_conv.names.COL_UE_BAUNIT_T: None,
+                  db_conv.names.LC_PARTY_T: None,
+                  db_conv.names.LC_PARTY_CONTACT_T: None,
+                  db_conv.names.LC_GROUP_PARTY_T: None,
+                  db_conv.names.MEMBERS_T: None,
+                  db_conv.names.LC_RIGHT_T: None,
+                  db_conv.names.LC_RESTRICTION_T: None,
+                  db_conv.names.LC_ADMINISTRATIVE_SOURCE_T: None,
+                  db_conv.names.LC_SPATIAL_SOURCE_T: None,
+                  db_conv.names.EXT_ARCHIVE_S: None,
+                  db_conv.names.COL_RRR_SOURCE_T: None,
+                  db_conv.names.COL_UNIT_SOURCE_T: None,
+                  db_conv.names.COL_UE_SOURCE_T: None,
+                  db_conv.names.COL_BAUNIT_SOURCE_T: None}
+
+        self.print_info(QCoreApplication.translate("DataModelConverterDialog",'Loading layers...'))
+        self.app.core.get_layers(db_pre, pre_layers, load=True)
+        self.app.core.get_layers(db_conv, conv_layers, load=True)
+
+        if not pre_layers or not conv_layers:
+            return
```

Comment:
The return of the function should be standard: 
e.g. https://github.com/Colsolutions/Asistente-LADM-COL/blob/b0a30c5303a9b5d2be1f20a4862906fd9b5f778e/asistente_ladm_col/gui/data_model_converter/dlg_data_model_converter.py#L336

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/577/commits/4888b2f347196a8dc65bad715dc7f8c71a5731dd